### DISCUSSION TITLE
[Feature Request] Github/Obsidian Alert type rendering

### DISCUSSION THREAD

**Discussion by lxymahatma** [CaiJimmy/hugo-theme-stack#1085] [Ideas]
Since Hugo's v0.133.0 add Github alert type support, and v0.134.0 add Obsidian alert type support. Documentation [here](https://gohugo.io/render-hooks/blockquotes/)

I think maybe we can have the alert type rendering support.
[Coder Theme](https://github.com/luizdepra/hugo-coder) uses shortcodes to support its custom rendering like `Note`, `Tip`, `Example` but using Github or Obsidian's alert style is much better

**Comment by cmpute** [comment]
https://zyxin.xyz/blog/tutorial/#提示框

我的主题fork里面已经支持了这一功能，如果觉得风格合适的话我可以把这个功能upstream上来。 @CaiJimmy

**Reply by cmpute** [reply]
<img width="1498" height="1202" alt="image" src="https://github.com/user-attachments/assets/0460f391-cd86-437a-b2e0-a37a964423c3" />

示意图⬆️
